packages/x-adapter-platform/src/endpoint-adapters/utils.ts

```diff
@@ -40,13 +40,32 @@ export function getBeaconServiceUrl(from: ExtraParamsRequest): string {
  * @internal
  */
 export function getQuestionsServiceUrl(from: ExtraParamsRequest): string {
+  if (from.extraParams?.env === 'test') {
+    return 'https://questions.internal.test.empathy.co'
+  }
+  if (from.extraParams?.env === 'staging') {
+    return 'https://questions.staging.empathy.co/v1/questions'
+  }
+  return 'https://api.{extraParams.env(.)}empathy.co/questions'
+}
+
+/**
+ * Gets the Tasks service URL for the given request.
+ *
+ * @param from - The request.t
+ *
+ * @returns The service URL.
+ *
+ * @internal
+ */
+export function getTasksServiceUrl(from: ExtraParamsRequest): string {
```

Comment:
Even they are in the same service.. Tasks urls are handled different from one env to another... so we create another method just to handle that issue..